obelisk_ws/src/robots/achilles_model/mujoco/achilles.xml

```diff
@@ -0,0 +1,303 @@
+<!-- <?xml version="1.0" encoding="utf-8"?> -->
+<mujoco model="achilles">
+
+  <!-- Set the angles to be in radians -->
+  <compiler angle="radian" autolimits="true" eulerseq="zyx" meshdir="./meshes"/>
+
+  <!-- Fix simulation options -->
+  <option timestep="0.001" iterations="50" solver="Newton" gravity="0 0 -9.81"  integrator = "RK4"/>
+
+  <default> <geom contype="0" conaffinity="0"/> </default>.
+
+  <size nconmax="100" njmax="500"/>
```

Comment:
both of these are [deprecated fields](https://mujoco.readthedocs.io/en/stable/XMLreference.html#size-nconmax) after version 2.3.0